packages/models-library/src/models_library/services_authoring.py

```diff
@@ -0,0 +1,47 @@
+from pydantic import BaseModel, Extra, Field, HttpUrl
+
+from .emails import LowerCaseEmailStr
+
+
+class Badge(BaseModel):
+    name: str = Field(
+        ...,
+        description="Name of the subject",
+        examples=["travis-ci", "coverals.io", "github.io"],
+    )
+    image: HttpUrl = Field(
+        ...,
+        description="Url to the badge",
+        examples=[
+            "https://travis-ci.org/ITISFoundation/osparc-simcore.svg?branch=master",
+            "https://coveralls.io/repos/github/ITISFoundation/osparc-simcore/badge.svg?branch=master",
+            "https://img.shields.io/website-up-down-green-red/https/itisfoundation.github.io.svg?label=documentation",
+        ],
+    )
+    url: HttpUrl = Field(
+        ...,
+        description="Link to the status",
+        examples=[
+            "https://travis-ci.org/ITISFoundation/osparc-simcore 'State of CI: build, test and pushing images'",
+            "https://coveralls.io/github/ITISFoundation/osparc-simcore?branch=master 'Test coverage'",
+            "https://itisfoundation.github.io/",
+        ],
+    )
+
+    class Config:
+        extra = Extra.forbid
```

Comment:
so do we want to follow the nice extra philosophy of not caring if there are additional keys and ignore them?

Reply:
IMO when it is an output, it should be strict (we define exactly what is provided) and when it is an input, it should be flexible (we take what we need and ignore the rest)